### PR TITLE
[Hackney] Small changes to support defect fetching

### PR DESCRIPTION
**Pull request by davea**
A couple of bug fixes and some customisation for the way Hackney defects are configured in Alloy.

### CODE REVIEW

File: perllib/Open311/Endpoint/Integration/AlloyV2.pm [hunk 1]
```diff
@@ -71,6 +71,11 @@ has config => (
     default => sub { $_[0]->alloy->config }
 );
 
+has group_in_service_code => (
```
Comment (dracos) [inline perllib/Open311/Endpoint/Integration/AlloyV2.pm:74]: If I've understood this right, should this not also have a change in post_service_request to not split the service_code on `_` into group/category? If it's more complicated, I think this needs a comment/documenting :)

File: perllib/Open311/Endpoint/Integration/AlloyV2.pm [hunk 2]
```diff
@@ -683,11 +683,20 @@ sub get_latlong_from_request {
...
-        $latlong = [ $points[$half]->[0], $points[$half]->[1] ];
+        $latlong = [ $points[$half]->[1], $points[$half]->[0] ];
+    } elsif ( $geometry->{type} eq 'MultiLineString') {
+        my @points;
+        my @segments = @{ $geometry->{coordinates} };
+        for my $segment (@segments) {
+            push @points, @$segment;
+        }
+        my $half = int( @points / 2 );
+        $latlong = [ $points[$half]->[1], $points[$half]->[0] ];
```
Comment (dracos) [inline perllib/Open311/Endpoint/Integration/AlloyV2.pm:706]: How come the Polygon $latlong line doesn't need updating as well?

Comment (davea) [reply perllib/Open311/Endpoint/Integration/AlloyV2.pm:706]: Who says it doesn't? 🤦 Thanks!

File: perllib/Open311/Endpoint/Integration/AlloyV2.pm (hunk 1 above)
Comment (davea) [reply perllib/Open311/Endpoint/Integration/AlloyV2.pm:74]: In actual fact `$group`/`$category` aren't used at all! So I'll get rid.

**Pull request merged**